src/sodium.iced

```diff
@@ -124,4 +124,23 @@ exports.Sodium = class Sodium extends Base
     if not opened then throw new Error('Sodium box_open_afternm failed!')
     else return opened
 
+  #
+  # @method scalarmult_base
+  # Compute the scalar product of a standard group element G and
+  # integer n on curve 25519. Returns resulting group element Q.
+  # @param {Buffer} n Integer.
+  # @return {Buffer} Resulting group element.
+  scalarmult_base : (n) ->
```

Comment:
@maxtaco  Let me know if it's fine to have arguments like that in there. I've done it like that because for scalarmult there would be no descriptive names like "secret" or "publicKey". The naming is consistent with http://nacl.cr.yp.to/scalarmult.html

I can also change this to use the "named arguments" like other functions, so it's a decision whether to stay faithful to underlying libraries.